rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/parsetreetoast/data_definitions.kt

```diff
@@ -184,7 +186,7 @@ internal fun RpgParser.Parm_fixedContext.toAst(
     }
 
     val baseType =
-        when (this.DATA_TYPE()?.text?.trim()?.toUpperCase()) {
+        when (this.DATA_TYPE()?.text?.trim()?.uppercase()) {
```

Comment:
Why do you change `toUpperCase()` in `uppercase`?

Reply:
Because toUpperCase is deprecated